hmcts/sds-flux-config
# DTSPO-6494 - fix dev config and tidy up

### Change description ###
Tested build, no changes found on ss-dev-01
- Add 00/01 overlays
- Remove duplicate identity patch


**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```
